pkg/services/ngalert/provisioning/alert_rules_test.go

```diff
@@ -52,6 +52,19 @@ func TestAlertRuleService(t *testing.T) {
 		require.Equal(t, interval, rule.IntervalSeconds)
 	})
 
+	t.Run("if a folder was renamed the interval should be fetched from the renamed folder", func(t *testing.T) {
```

Comment:
We should add another copy of this test, but renaming the `RuleGroup` field rather than `NamespaceUID` as I believe the bug exists there too.

Reply:
This cannot happen, as the rule group is not editable through the UI. The Folder is an exception here.